test/Confluent.Kafka.IntegrationTests/Tests/Consumer_Poll.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2016-2017 Confluent Inc.
```

Comment:
And yeah, I've heard of namespaces and folders. 

We're starting to get enough tests that we need to start structuring them a bit more. I'll try to avoid doing this.